Contribute/index.md

```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
 
 Welcome to the [docs.microsoft.com](https://docs.microsoft.com) (Docs) Contributor Guide!
 
-## About this guide
+Several of our documentation sets are Open Source, hosted on GitHub. More teams are adopting this model all the time. It streamlines and improves communication between the product  engineers, the content teams, and our customers. Working in the open provides several advantages:
```

Comment:
There's an extra space between "product" and "engineers".

Reply:
Can we add a sentence that says something like "Even document sets that are not completely open source have public-facing repos where you are invited to make pull requests."